charts/rancher-backup/values.yaml

```diff
@@ -90,3 +90,19 @@ monitoring:
     additionalLabels: {}
     metricRelabelings: []
     relabelings: []
+  
+  alerts:
+    enabled: false
+
+    ## Define custom alerting rules here.
+    ## The "BackupFailed" alert is included by default when .Values.monitoring.alerts.enabled is set to true and rancher-monitoring is installed.
+    alertingRules:
```

Comment:
I would consider renaming this to prometheus rules or something like that to be able to declare both alerting and generic prometheus rules